app/src/main/java/org/wikipedia/feed/news/NewsFragment.java

```diff
@@ -166,6 +168,12 @@ public void onAddPageToList(@NonNull HistoryEntry entry) {
                     AddToReadingListDialog.newInstance(entry.getTitle(), NEWS_ACTIVITY));
         }
 
+        @Override
+        public void onMovePageToList(long sourceReadingListId, @NonNull HistoryEntry entry) {
+            bottomSheetPresenter.show(getChildFragmentManager(),
+                    MoveToReadingListDialog.newInstance(sourceReadingListId, entry.getTitle(), MOST_READ_ACTIVITY));
```

Comment:
Is `MOST_READ_ACTIVITY` the correct constant here?

Reply:
It is not. Updated and done!